File: Crio/crio_programming_interview_problems-master/ContainsDuplicate/ContainsDuplicate.cpp
#include <bits/stdc++.h>
#include "FastIO.hpp"
#include "ReadMatrix.hpp"
using namespace std;

class ContainsDuplicate {
public:
    bool containsDuplicate(vector<int>& nums) {
    	// CRIO_SOLUTION_START_MODULE_L1_PROBLEMS
        sort(nums.begin(),nums.end());
        for(int i=0;i<int(nums.size()-1);i++)
            if(nums[i] == nums[i+1]) return true;
        return false;
        // CRIO_SOLUTION_END_MODULE_L1_PROBLEMS
    }
};

int main() {
	FastIO();
	int n;
	cin >> n;
	vector<int> nums;
	ReadMatrix<int>().OneDMatrix(n, nums);
	bool result = ContainsDuplicate().containsDuplicate(nums);
	cout << boolalpha << result;
	return 0;
}

File: Crio/crio_programming_interview_problems-master/ContainsDuplicate/ContainsDuplicate.py
def containsDuplicate(nums):
    nums = sorted(nums)
    for i in range(len(nums)-1):
        if nums[i]==nums[i+1]:
            return True

    return False

if __name__ == '__main__':
    n = int(input())
    nums = []
    if n > 0:
        nums = input().split()

    nums = [int(i) for i in nums]
    result = containsDuplicate(nums)
    if result:
        print('true')
    else:
        print('false')
